R/tfrmt.R

```diff
@@ -21,6 +21,7 @@
 #' @param col_plan a col_plan object which is used to select, rename, and nest
 #'   columns. Takes a [col_plan()]
 #' @param sorting_cols which columns determine sorting of output
+#' @param big_n Takes a [big_n_structure()] to select with rows of the ARD should go into the column labels as big N's
```

Comment:
In comparison to the other param descriptions, this seems more focused on what is happening behind the scenes (i.e. selecting rows). I might suggest revising to be more about the end goal:

'how to format subject totals ("big Ns") for inclusion in the column labels. Takes a big_n_structure()'